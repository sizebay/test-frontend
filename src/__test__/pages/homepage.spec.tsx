import { render } from "@testing-library/react";
import Page from "@/app/(home-page)/page";

it("Renders homepage", () => {
  const { container } = render(<Page />);
  expect(container).toMatchSnapshot();
});
