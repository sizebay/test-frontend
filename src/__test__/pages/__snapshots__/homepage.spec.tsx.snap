// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Renders homepage 1`] = `
<div>
  <main
    class="h-screen w-screen bg-background"
  >
    <div
      class="flex flex-col w-full h-full items-center justify-start py-6 px-2 sm:px-0 mx-auto max-w-full sm:max-w-5/6 lg:max-w-5xl"
    >
      <h2
        class="text-4xl text-foreground"
      >
        Hello World
      </h2>
    </div>
  </main>
</div>
`;
